pyproject.toml

```diff
@@ -37,22 +37,29 @@ dependencies = [
     "firerequests>=0.1.2",
     "numpy>=2.0.2",
     "supervision>=0.25.1",
+    "validators>=0.34.0",
 ]
 
 [project.optional-dependencies]
+
 cpu = [
     "torch>=2.6.0",
     "torchvision>=0.21.0",
 ]
+
 cu124 = [
     "torch>=2.6.0",
     "torchvision>=0.21.0",
 ]
 
+rocm61 = [
+    "torch",
+    "torchvision",
+]
```

Comment:
@onuralpszr Should we pin the torch versions for rocm?

Reply:
@soumik12345 I am going to check min py3.9 for all and others then we can pin "min" for all

Reply:
I added min for all.